src/speechbox/__init__.py

```diff
@@ -28,5 +28,6 @@
 
 if is_transformers_available() and is_accelerate_available() and is_scipy_available():
     from .restore import PunctuationRestorer
+    from .diarize import ASRDiarizationPipeline
```

Comment:
```suggestion
if is_transformers_available() and is_accelerate_available() and is_torchaudio_available() and is_pyonnote_available():
    from .diarize import ASRDiarizationPipeline
```

Reply:
Note that you should create the `is_torchaudio_available()` and `is_pyannote_available()` functions.

Reply:
Then you should put it under a new if statement and run:

```
make fix-copies
```

and dummy objects will be automatically created

Reply:
Resolved in https://github.com/huggingface/speechbox/pull/9/commits/c483e244cd92afaa4e39c8862a7068808cac9d8a